awshelper/config_test.go

```diff
@@ -36,9 +38,61 @@ func TestAwsIsAddedInUserAgent(t *testing.T) {
 func TestAwsSessionValidationFail(t *testing.T) {
 	t.Parallel()
 
-	err := awshelper.ValidateAwsSession(&awshelper.AwsSessionConfig{
+	session, err := awshelper.CreateAwsSession(&awshelper.AwsSessionConfig{
```

Comment:
NIT: Try not to change the names of variables like this if you can. We pass around `sess` as the abbreviation of `session` in calls, so let's try to always call it `sess`, even here.

Reply:
Fixed